src/demo/website/Mybuild

```diff
@@ -11,8 +11,8 @@ module website {
 		"joke.html",
 		"matmex.gif",
 		"mm_en.gif",
-		"test_temp.html"
-
+		"test_temp.html",
+		"test"
```

Comment:
Why do you use website for adding test file for rootfs? You should use `@InitFS` annotation instead